manual_test_websocket/README.md

```diff
@@ -26,28 +26,34 @@ To start AWS DynamoDB locally (can run only after first deploying locally): `sls
 
 ## Testing on AWS
 
-`npm --endpoint={WebSocket endpoint URL on AWS} run test`
+`npm --endpoint={WebSocket endpoint URL on AWS} --timeout={timeout in ms} run test`
 
 
 ## Usage Assumption - In order to send messages back to clients
-`const newAWSApiGatewayManagementApi=(event, context)=>{`
+ 
+`POST http://localhost:3001/@connections/{connectionId}`
 
-`  const endpoint=event.requestContext.domainName+'/'+event.requestContext.stage;`
+Or,
 
-`  const apiVersion='2018-11-29';`
+`const endpoint=event.requestContext.domainName+'/'+event.requestContext.stage;`
 
-`  let API=context.API;`
+`const apiVersion='2018-11-29';`
 
-`  if (!process.env.IS_OFFLINE) {`
+`let API=null;`
 
-`    API = require('aws-sdk');`
+`if (!process.env.IS_OFFLINE) {`
 
-`    require('aws-sdk/clients/apigatewaymanagementapi');`
+`  API = require('aws-sdk');`
 
-`  }`
+`  require('aws-sdk/clients/apigatewaymanagementapi');`
```

Comment:
haven't looked at the entirety, but is this `require` needed for anything?

Reply:
`require('aws-sdk/clients/apigatewaymanagementapi')` is a know workaround to an AWS SDK limitation.
Nevertheless, I tried to test without this `require` and it runs without any problem. They must have recently fixed the issue. I'll update code and documentation.

Reply:
Removed from code and docs.